components/CurrencyCard.tsx

```diff
@@ -0,0 +1,69 @@
+import { StyleSheet, Text, View, Image } from 'react-native';
+import React, {FC, ReactElement, ReactNode} from 'react';
+
+import Icon from 'react-native-vector-icons/Feather'
+Icon.loadFont()
+
+const CurrencyCard = ({currency}: any) => {
+  
+
+  return (
+    <View style={styles.mainContainer}>
+      <View style={styles.leftContainer}>
+        <Image style={styles.icon} source={{uri: `https://messari.io/asset-images/${currency.id && currency.id}/128.png?v=2`}} />
+        <View>
+          <Text style={styles.boldText}>{currency.name && currency.name}</Text>
+          <Text>{currency.symbol && currency.symbol}</Text>
+        </View>
+      </View>
+      <View>
+        <Text style={[styles.boldText]}>{`$${currency.price && currency.price.toFixed(0)}`}</Text>
+        <View style={styles.percent}>
+          {(currency.pctg && currency.pctg >= 0) ? <Icon name="arrow-up-right" size={17} color="green" /> : <Icon name="arrow-down-left" size={17} color= 'red' />}
+          <Text style={[styles.span, {color: (currency.pctg && currency.pctg >= 0) ? 'green' : 'red'}]}>
```

Comment:
[suggestion] For better code readability, it would be better in this case to create a variable, e.g. "isPositive" to validate against this.